docs/src/main/asciidoc/security-keycloak-admin-client.adoc

```diff
@@ -85,16 +87,72 @@ public class RolesResource {
 }
 ----
 
-Now, you can produce the `Keycloak` bean yourself, or you can instruct Quarkus to produce `Keycloak` bean.
+<1> Create  a default Keycloak Admin Client which can perform Keycloak `master` realm administration tasks as an `admin-cli` client, such as adding new realms, clients and users.
```

Comment:
Can you stick the callout to the `----` by removing the new line? I'm not sure if all Asciidoc impl allows for the new line.

Reply:
Sure